demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,25 @@
+/*
+#!/usr/bin/env node
+*/
+/* eslint-disable require-jsdoc */
+
+class ArticleRef {
+  constructor(lang, title, revision) {
+    this.lang = lang
+    this.title = title
+    this.revision = revision
+  }
+  fileName() {
+    return `${this.lang}.${this.title}.${this.revision}.json`
```

Comment:
Should we include the revision in the filename? If we ever change the revision, we'll see a new file generated and the old one will be unchanged and not identified by Git.